src/content/ci/github-actions.mdx

```diff
@@ -496,11 +496,13 @@ Read the official [Storybook environment variable's documentation](https://story
 
 ### Recommended configuration for build events
 
-GitHub's Actions, like other CI systems, can run based on any type of event. Our recommendation is to run Chromatic's step on `push` events. All other event types except `pull-request` will not work.
+GitHub's Actions, like other CI systems, can run based on any type of event. Our recommendation is to run Chromatic's step on `push` events.
 
-The `pull-request` event requires special consideration. Like other CI systems, GitHub allows workflow execution on either commits pushed to a branch in a pull request. Or for "merge" commits between that branch and the base branch (main).
+All other event types except `pull_request` will not work. However, using the `pull_request` event might cause Chromatic's baselines to be lost in certain scenarios.
 
-These specific types of commits (merge) don't persist in the history of your repository. That can cause Chromatic's baselines to be lost in certain situations. Hence why we recommend running Chromatic's step on `push`.
+GitHub allows workflows to execute on commits pushed to a branch in a pull request or on "merge" commits between that branch and the base branch (main). These merge commits do not persist in your repository's history. When a GitHub action is triggered on `pull_request`, GitHub creates an ephemeral branch and merges the PR branch into the latest from `main`. For details, refer to the [pull request workflow docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) to understand what `GITHUB_SHA` and `GITHUB_REF` are set to.
+
+This means the latest build on main will be an ancestor of this ephemeral branch, even though it is not an ancestor of the actual PR branch. Therefore, it'll use the incorrect baseline for comparison. That's why we recommend running Chromatic using the `push` event.
```

Comment:
Do we also want to add a bit about the workaround, which is to ensure you're checking out the actual branch when using `actions/checkout`? This bit:

```
- uses: actions/checkout@v3
  with:
    fetch-depth: 0
    # 👇 Tells the checkout which commit hash to reference
    ref: ${{ github.event.pull_request.head.ref }}
```

Reply:
I was thinking no. We don't actually want people to use `pull_request` so maybe not even mention this. WDYT?

Reply:
Sounds good. To confirm, they should be using this in all steps of their workflow?

Reply:
That's a good question. I'd say no, only when doing a chromatic build. But it does bring up a good point. If people have one big workflow that checks out code and does a bunch of other things alongside building chromatic, then that change will affect the other non-chromatic things. Hmmmm

Reply:
@tevanoff okay, pushed an update. Let me know what you think